src/test/java/teammates/test/util/Url.java

```diff
@@ -0,0 +1,16 @@
+package teammates.test.util;
+
+import teammates.test.driver.TestProperties;
+
+public class Url extends teammates.common.util.Url {
```

Comment:
Sure. 
May be have a `Url` class and two sub classes `AppUrl` and `TestAppUrl`(?)